web/cobrands/smidsy/js/report.js

```diff
@@ -50,4 +50,40 @@ $(function() {
         }
     });
 
+    // Deal with toggling stats19 data on and off on the /around page
+    var $stats19Link = $('#stats-19');
+    var showText = 'Show reports from the Department of Transport';
+    var hideText = 'Hide reports from the Department of Transport';
+    var toggleStats19Link = function toggleStats19Link($link){
+      if ($link.data('show-stats19')) {
+        $link.data({'show-stats19': 0});
+        $link.html(hideText);
+      } else {
+        $link.data({'show-stats19': 1});
+        $link.html(showText);
+      }
+    };
+    if (window.fixmystreet.show_stats19 === '1') {
+      // Force a load of the pins with the stats_19 param set up on first
+      // load because FMS' default _onload function won't know about it.
+      window.fixmystreet.markers.protocol.options.params.show_stats19 = '[% show_stats19 %]';
```

Comment:
woops, ta - I had this elsewhere in a template initially, obviously didn't check when I moved it.. Fixed in eb129b360821a50266a1a8092c4265dab68e0188